package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "build": "bsb -make-world",
     "start": "bsb -make-world -w",
-    "clean": "bsb -clean-world"
+    "clean": "bsb -clean-world",
+    "test": "jest --watch"
```

Comment:
Using `jest --watch` does not seem to work for me. We could just use `jest`, which is probably more what we'll usually want anyways, and gives us a target to run in CI. It gives the following error:
```
Error: watch /home/zach/src/bs-puppeteer ENOSPC
    at _errnoException (util.js:1022:11)
    at FSWatcher.start (fs.js:1374:19)
    at Object.fs.watch (fs.js:1400:11)
    at NodeWatcher.watchdir (/home/zach/src/bs-puppeteer/node_modules/sane/src/node_watcher.js:175:20)
    at new NodeWatcher (/home/zach/src/bs-puppeteer/node_modules/sane/src/node_watcher.js:45:8)
    at createWatcher (/home/zach/src/bs-puppeteer/node_modules/jest-haste-map/build/index.js:572:23)
    at Array.map (<anonymous>)
    at HasteMap._watch (/home/zach/src/bs-puppeteer/node_modules/jest-haste-map/build/index.js:683:44)
    at _buildPromise._buildFileMap.then.then.hasteMap (/home/zach/src/bs-puppeteer/node_modules/jest-haste-map/build/index.js:280:21)
    at <anonymous>
```

Reply:
`Error: watch /home/zach/src/bs-puppeteer ENOSPC`
It seems like you're running out of space.

However, I've changed to jest for `test` and added `test:watch` for watch mode: . (ref.: a3fad57)